packages/opentelemetry-plugin-grpc/src/types.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as grpcModule from 'grpc';
+import * as events from 'events';
+
+export type grpc = typeof grpcModule;
+
+export type SendUnaryDataCallback = (
+  error: grpcModule.ServiceError | null,
+  // tslint:disable-next-line:no-any
+  value?: any,
+  trailer?: grpcModule.Metadata,
+  flags?: grpcModule.writeFlags
+) => void;
+
+export interface GrpcPluginOptions {}
+
+interface GrpcStatus {
+  code: number;
+  details: string;
+  metadata: grpcModule.Metadata;
+}
+
+export type ServerCall =
+  | typeof grpcModule.ServerUnaryCall
+  | typeof grpcModule.ServerReadableStream
+  | typeof grpcModule.ServerWritableStream
+  | typeof grpcModule.ServerDuplexStream;
+
+export type ServerCallWithMeta = ServerCall & {
+  metadata: grpcModule.Metadata;
+  status: GrpcStatus;
+  // tslint:disable-next-line:no-any
+  request?: any;
```

Comment:
Would it work for this to be of `unknown` type?

Reply:
Changed to `unknown` type (and a few other places where it made sense)